src/@ionic-native/plugins/firebase-lib/index.ts

```diff
@@ -0,0 +1,369 @@
+import { Injectable } from '@angular/core';
+import { Cordova, IonicNativePlugin, Plugin } from '@ionic-native/core';
+import { Observable } from 'rxjs/Observable';
```

Comment:
to use this plugin sucessfully I had to use this instead.

import { Observable } from 'rxjs';